src/toolchains.ts

```diff
@@ -0,0 +1,133 @@
+import * as path from 'path';
+import * as core from '@actions/core';
+import * as io from '@actions/io';
+
+import * as fs from 'fs';
+import * as os from 'os';
+
+import { create as xmlCreate } from 'xmlbuilder2';
+import * as constants from './constants';
+import { getBooleanInput } from './util';
```

Comment:
Unfortunately we don't really have a consistent import style in this codebase.  Let's just go ahead and group them in a way where it's clear where to insert new ones.

```suggestion
import * as fs from 'fs';
import * as os from 'os';
import * as path from 'path';
import * as core from '@actions/core';
import * as io from '@actions/io';
import * as constants from './constants';

import { getBooleanInput } from './util';
import { create as xmlCreate } from 'xmlbuilder2';
```

Reply:
Fixed